ssh_check/test/test_ssh.py

```diff
@@ -57,7 +57,6 @@ def test_ssh(self):
 
         service = self.check.get_service_checks()
         self.assertEqual(service[0].get('status'), AgentCheck.OK)
-        self.assertEqual(service[0].get('message'), None)
```

Comment:
can we assert message is `No errors occured`?

Reply:
👍